runtime/parser/declaration_test.go

```diff
@@ -2557,6 +2557,150 @@ func TestParseEvent(t *testing.T) {
 		)
 	})
 
+	t.Run("default event", func(t *testing.T) {
+
+		t.Parallel()
+
+		result, errs := testParseDeclarations(` access(all) event ResourceDestroyed ( a : String = "foo")`)
+		require.Empty(t, errs)
+
+		utils.AssertEqualWithDiff(t,
+			[]ast.Declaration{
+				&ast.CompositeDeclaration{
+					Members: ast.NewUnmeteredMembers(
+						[]ast.Declaration{
+							&ast.SpecialFunctionDeclaration{
+								FunctionDeclaration: &ast.FunctionDeclaration{
+									ParameterList: &ast.ParameterList{
+										Parameters: []*ast.Parameter{
+											{
+												TypeAnnotation: &ast.TypeAnnotation{
+													Type: &ast.NominalType{
+														Identifier: ast.Identifier{
+															Identifier: "Int",
```

Comment:
The report from `assert.Equal` is actually correct, and `deep.Equal` is not properly reporting the difference here, and for the position info

Reply:
I though I fixed this somewhere 🤔 
Let me double check